bin/read-google-sheets-moc-v2.js

```diff
@@ -0,0 +1,74 @@
+const google = require('googleapis');
+const keys = require('../creds-google-sheet.json');
+const googleMethods = require('../server/recess-events/google-methods');
+const firebasedb = require('../server/lib/setupFirebase.js').realtimedb;
+
+const SHEETS_ID = process.env.SHEETS_ID_MOC_EB_IDS;
+const SCOPES = ['https://www.googleapis.com/auth/spreadsheets'];
+
+// TODO: change this to actual path
+const ROOT_PATH_MOC = 'test_moc';
+
+// TODO: move this into models script
```

Comment:
Personally, since this is only being used here, I don't mind that it stays here. In fact, I think it probably should, since the fields are kind of specific to this use case. (If we needed a general purpose class, we'd probably want to represent the congress number better.)